config/management/operational/src/network_checker.rs

```diff
@@ -83,8 +83,8 @@ impl CheckEndpoint {
 #[cfg(test)]
 pub mod tests {
     use crate::test_helper::OperationalTool;
-    use libra_network_address::NetworkAddress;
-    use libra_types::chain_id::ChainId;
+    use diem_network_address::NetworkAddress;
+    use diem_types::chain_id::ChainId;
```

Comment:
@JoshLind I had to keep the `libra` and `libra.org` names below for the network addreses. Just want to make sure this is OK/that you're aware.

Reply:
@tzakian, this should be fine -- we can clean it up later 😄 Out of curiosity, did the test fail if you changed them?

Reply:
Yea, sadly the test failed if they are changed :( So figured best to switch them back for now :)

Reply:
Awesome, thanks! 😄